activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeBPMNMessageIT.java

```diff
@@ -727,4 +789,57 @@ public void shouldTestBoundaryMessageExpressionWithNotFoundMessageEventSubscript
         assertThat(thrown).isInstanceOf(ActivitiObjectNotFoundException.class);
     }  
     
+    @Test
+    public void should_getMessageCancelledEvent_when_processIsEnded() {
+
+        securityUtil.logInAs("user");
+
+        ProcessInstance process = processRuntime.start(ProcessPayloadBuilder.start()
+                                                       .withBusinessKey("businessKey")
+                                                       .withVariable("correlationKey", "correlationKey")
+                                                       .withProcessDefinitionKey(CATCH_MESSAGE)
+                                                       .build());
+        
+        processRuntime.delete(ProcessPayloadBuilder.delete(process.getId()));    
+        
+        assertThat(MessageTestConfiguration.messageEvents).isNotEmpty()
```

Comment:
I'd move this assert before deleting the process, as MESSAGE_WAITING is produced by the start process.